src/datachain/lib/settings.py

```diff
@@ -66,3 +74,5 @@ def add(self, settings: "Settings"):
         self.parallel = settings.parallel or self.parallel
         self._workers = settings._workers or self._workers
         self.min_task_size = settings.min_task_size or self.min_task_size
+        if settings.prefetch is not None:
```

Comment:
is a reason to have a mix of styles here - some protected vars, some not, some ` self._cache = settings._cache or self._cache`, some like `if settings.prefetch is not None:`?

Reply:
I don't see the point of using private attributes, but I didn't want to refactor `Settings` as that isn't directly related to this PR.
Using `settings.prefetch or self.prefetch` would be incorrect, as both `0` and `None` are valid values with different meanings (`0` means "disable prefetching", `None` means "use the default value").